sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 
 ## 1.0.0-beta.5 (Unreleased)
+### Breaking Changes
+
+- ChatMessage - `senderId` renamed to `senderCommunicationIdentifier`, changed type to `CommunicationIdentifier`.
```

Comment:
Instead of CommunicationIdentifier, which can be quite verbose, what about just "Identifer"?

Reply:
That's the literal name of the class

Reply:
This will get renamed likely in the next round, so we can tackle it there with the other sdks.